### ISSUE TITLE
request: Speed improvements to dev workflow

### ISSUE THREAD

**Issue by hbons** [LIJI32/SameBoy#497]
I'm creating a small GameBoy game as a side-project on macOS.

I use `make run` to compile and start my game, which launches it in SameBoy using `open -a SameBoy obj/GAME.gb`
The problem is that when I forget to close the emulation window, restarting this command will only refocus SameBoy.app and not reload the rom. Pressing CMD+R only reloads the rom from memory and not from the filesystem.

To increase the speed of my workflow, is it possible to force reload from filesystem using a command line argument or something? If the file ID of the rom isn't tracked, just being able to close all current emulation windows from the command line would help too. I couldn't find any documentation on this.

Additionally being able to set an argument to skip the GameBoy logo splash screen would speed things up too.

**Comment by LIJI32** [comment]
CMD+R should actually reload the ROM file (together with symbols) from the filesystem and not memory. Are you sure about that?

**Comment by hbons** [comment]
You're right, it does reload from filesystem properly. Automating the reload would still be nice though. :)
